compiler/noirc_frontend/src/hir/comptime/interpreter/builtin.rs

```diff
@@ -684,21 +686,20 @@ fn quoted_as_expr(
 ) -> IResult<Value> {
     let argument = check_one_argument(arguments, location)?;
 
-    let result =
-        parse(interner, argument.clone(), Parser::parse_expression_or_error, "an expression");
-    if let Ok(expr) = result {
-        return option(return_type, Some(Value::expression(expr.kind)));
-    }
-
-    let result =
-        parse(interner, argument.clone(), Parser::parse_statement_or_error, "an expression");
-    if let Ok(stmt) = result {
-        return option(return_type, Some(Value::statement(stmt.kind)));
-    }
+    let result = parse(
+        interner,
+        argument.clone(),
```

Comment:
```suggestion
        argument,
```
doesn't look like you need the clone here